btx/processing/merge.py

```diff
@@ -0,0 +1,262 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import argparse
+import os
+import requests
+
+class StreamtoMtz:
+    
+    """
+    Wrapper for calling various CrystFEL functions for generating a mtz
+    file from a stream file and reporting statistics. The output files,
+    including hkl, mtz, dat, and figures, inherit nomenclature from the
+    input stream file.
+    """
+    
+    def __init__(self, input_stream, symmetry, taskdir, cell):
+        
+        self.stream = input_stream # file of unmerged reflections, str
+        self.symmetry = symmetry # point group symmetry, str
+        self.taskdir = taskdir # path for storing output, str
+        self.cell = cell # pdb or CrystFEL cell file, str
+        self._set_up()
+        
+    def _set_up(self):
+        """
+        Retrieve number of processors to run partialator and the temporary
+        file to which to write the command to sbatch.
```

Comment:
Do we want to check that `partialator`, `compare_hkl`, `get_hkl` can be found and throw an error if that's not the case? Might help troubleshoot?

Reply:
Done using `shutil.which` functionality.